products/appengine/api.yaml

```diff
@@ -446,3 +446,42 @@ objects:
           Instance class that is used to run this version. Valid values are
           AutomaticScaling F1, F2, F4, F4_1G
           (Only AutomaticScaling is supported at the moment)
+  - !ruby/object:Api::Resource
+    name: 'ApplicationUrlDispatchRules'
```

Comment:
We need to add an `async` block so that Terraform knows to wait on async operations from the API; https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/appengine/api.yaml#L41-L57 contains an example.

I receive this error otherwise:

```
---[ RESPONSE ]--------------------------------------
HTTP/2.0 409 Conflict
Alt-Svc: quic=":443"; ma=2592000; v="46,43",h3-Q048=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000
Cache-Control: private
Content-Type: application/json; charset=UTF-8
Date: Mon, 14 Oct 2019 19:47:55 GMT
Server: ESF
Vary: Origin
Vary: X-Origin
Vary: Referer
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 0

{
  "error": {
    "code": 409,
    "message": "Cannot operate on apps/my-project because an operation is already in progress for apps/my-project by 9152be65-8e22-417e-a0d7-3c4151befddb.",
    "status": "ABORTED"
  }
}

```

Reply:
Could you please review the async block. I really doesnt have idea about path & base_url.

Reply:
It should be good, they're generally common inside products (and most are common across most products). Generating now to make sure.

Reply:
Thank you.  I was able to test this locally. Please let me know if your test is working. Thanks for your help !